microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,114 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+<!-- ------------------------------ -->
+#### Account-specific search functions
 
 Account-specific search functions are available if you are signed in to Microsoft Edge with a Microsoft work or school account, and Microsoft Search is available. Microsoft may send an anonymous token with your query, to provide account-specific functions, such as results specific to your company.
 
 All data is transmitted securely over HTTPS.  If [Bing](https://bing.com) is your default search provider, the searches and typed characters are saved for up to six months.
 
-If you search for a single word in the address box, Microsoft Edge may send the single word to your DNS server. Sending a single word is a check to see whether it corresponds to a host on your network. If so, Microsoft Edge may try to connect to the corresponding host. This option lets you go to the specific host instead of searching.  For example, if your router uses the hostname `router` and you type `router` in the address bar, you can go to `https://router`, or search for the word `router`.  The feature is not controlled by the **Show me search and site suggestions using my typed characters** setting because it does not involve sending data to your default search engine.
 
-You can control whether or not typed characters are sent to your default search provider. Go to `edge://settings/search`. Toggle the **Show me search and site suggestions using my typed characters** setting.
+<!-- ------------------------------ -->
+#### Single-word search
+
+If you search for a single word in the Address bar, Microsoft Edge may send the single word to your DNS server. Sending a single word is a check to see whether it corresponds to a host on your network. If so, Microsoft Edge may try to connect to the corresponding host. This option lets you go to the specific host instead of searching.  For example, if your router uses the hostname `router` and you type `router` in the Address bar, you can go to `https://router`, or search for the word `router`.  This feature isn't controlled by the **Show me search and site suggestions using my typed characters** setting, because this feature doesn't involve sending data to your default search engine.
 
 <!-- The Search settings seem different from what is described here -->
 
-You can change your default search engine. Go to `edge://settings/search`.  Select the **Search engine used in the address bar** dropdown menu.  If you are browsing while using **InPrivate** or **Guest** mode, your autosuggestions are turned off.  **InPrivate** shows suggestions from your local browsing, such as browsing history and past searches. No typed characters are sent to your default search engine.  **Guest** mode does not display any suggestions or send typed characters to your default search engine.
 
-Data collected by other search providers follow the privacy policy of the company.
+<!-- ------------------------------ -->
+#### Other search providers, InPrivate, and Guest mode
```

Comment:
changing your default search provider and how address bar is affected by inprivate / guest mode should be two different sections.

Reply:
@apryzgoda i moved the "Data that's" sentence from end of h2 up into "Changing default search provider" h4 section.  we can do further fine tuning of h4s structuring in future PRs.

i think this was my biggest question about my breakout of the h4 sections (not that i've closely analyzed every sentence or point in the entire h2 section).  was wondering about having two very short sections here.  

result:
```
<!-- ------------------------------ -->
#### Changing your default search provider

To change your default search engine, go to `edge://settings/search`, and then use the **Search engine used in the address bar** dropdown list.

Data that's collected by other search providers follows the privacy policy of that company.


<!-- ------------------------------ -->
#### How the Address bar is affected by InPrivate and Guest mode

If you're browsing while using **InPrivate** or **Guest** mode, your autosuggestions are turned off.

**InPrivate** shows suggestions from your local browsing, such as browsing history and past searches.  When using **InPrivate**, no typed characters are sent to your default search engine.

**Guest** mode doesn't display any suggestions or send typed characters to your default search engine.
```